src/modules/form/components/viewable/DynamicViewFields.tsx

```diff
@@ -113,21 +103,6 @@ const DynamicViewFields: React.FC<DynamicViewFieldsProps> = ({
   const definition = handlers.definition;
   const values = handlers.methods.getValues();
 
-  const itemChanged: ItemChangedFn = useCallback(
-    ({ linkId, value: baseValue, type }) => {
-      const item = handlers.itemMap[linkId];
-      let value = baseValue;
-      if (item) {
-        if (item.type === ItemType.Integer) value = parseInt(value) || 0;
-        if (item.type === ItemType.Currency) value = parseFloat(value) || 0;
```

Comment:
This is a little concerning because I don't know why this was here. I can pretty clearly trace that `itemChanged` was only used to adjust pick list option values, so this seems irrelevant.

Reply:
@gigxz look at this again. are there other components that were getting `itemChanged` and using it, like DisabilityTable

Reply:
looks like it was just duplicated from DynamicFormFields